test/convenience/webhook.test.ts

```diff
@@ -138,4 +138,24 @@ describe("webhook", () => {
             return handler(req);
         });
     });
+
+    it("bot should catch errors in webhoook handler", async () => {
+        bot.catch((err) => {
+            assertIsError(err, BotError, "Test Error");
+        });
```

Comment:
Can you assert that this handler is actually executed? Otherwise `assertIsError` might just never be called :D